### PR TITLE
Give the option to plot multiple polygons

### PR DESCRIPTION
**Pull request by loerum**
This commit will add the option to plot several polygons with different colors on the overpass drawings.
At DMI we have two areas of interest (Europe and Greenland).

### CODE REVIEW

File: trollsched/drawing.py [hunk 1]
```diff
@@ -176,10 +184,14 @@ def save_fig(pass_obj,
     plot_parameters = plot_parameters or {}
     with Mapper(**plot_parameters) as mapper:
         mapper.nightshade(pass_obj.uptime, alpha=0.2)
+        for i, p in enumerate(poly):
+            try:
+                c = poly_color[i]
```
Comment (adybbroe) [inline trollsched/drawing.py:189]: pep8 requires three letters for variables, did you run pylint @loerum ?
Suggest using `col` instead of `c`!

**Comment by adybbroe** [comment]
And by the way, wat about a unit test @loerum ?

File: trollsched/drawing.py (hunk 1 above)
Comment (adybbroe) [reply trollsched/drawing.py:189]: Oh, I see that we have a linter service here, sorry! Strange I thought it should have complained here... ?

Comment (loerum) [reply trollsched/drawing.py:189]: Hi Adam, I first saw your comments now (don't get notifications). My Python Flymake, does not complain about one letters variables. If it's a simple loop i don't mind one letters, but I will change.

Comment (loerum) [reply trollsched/drawing.py:189]: Hmm, you comment was it only about the variable c and not i and p ?

File: trollsched/drawing.py [hunk 2]
```diff
@@ -163,10 +171,9 @@ def save_fig(pass_obj,
     if not os.path.exists(directory):
         logger.debug("Create plot dir " + directory)
         os.makedirs(directory)
-    filename = os.path.join(
-        directory,
-        (rise + '_' + pass_obj.satellite.name.replace(" ", "_") + '_' + pass_obj.instrument.replace("/", "-") + '_' + fall + extension))
-
+    filename = os.path.join(directory, (rise + '_' +
```
Comment (sfinkens) [inline trollsched/drawing.py:174]: You could use `'{rise}_{sat_name}...{extension}'.format(**components)` here to increase readability.

Comment (adybbroe) [reply trollsched/drawing.py]: Fixed, thx

**Pull request merged**